scipy/stats/tests/generate_fisher_exact_results_from_r.R

```diff
@@ -0,0 +1,82 @@
+# This R script generates the Python file fisher_exact_results_from_r.py.
```

Comment:
Copyright header?

Reply:
I wrote the script, so it is under the same license as all of the developer contributions to SciPy.  We generally don't add copyright notices to our code.